x-pack/plugins/profiling/public/views/no_data_view/index.tsx

```diff
@@ -42,8 +42,8 @@ export function NoDataView({ subTitle }: { subTitle: string }) {
   );
 
   const secretToken = data?.variables.collector.secretToken;
-  const collectionAgentHost = data?.variables.collector.host.replace('https://', '');
-  const symbolUrl = data?.variables.symbolizer.host.replace('https://', '');
+  const collectionAgentHost = data?.variables.collector?.host?.replace('https://', '');
+  const symbolUrl = data?.variables?.symbolizer?.host?.replace('https://', '');
```

Comment:
The symbols URL is actually meant to contain the `https://` prefix, we can remove the `replace()`.
@rockdaboot @athre0z can you please confirm?

Reply:
Then I think we need this

```suggestion
  const symbolUrl = data?.variables?.symbolizer?.host?;
```

Reply:
Done!